src/main/scala/uk/gov/hmrc/api/filters/CacheControlFilter.scala

```diff
@@ -1,70 +0,0 @@
-/*
- * Copyright 2021 HM Revenue & Customs
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package uk.gov.hmrc.api.filters
-
-import akka.stream.Materializer
-import play.api.Play
-import play.api.Play.current
-import play.api.http.HttpVerbs.{GET => GET_METHOD}
-import play.api.mvc.{Filter, RequestHeader, Result}
-import play.mvc.Http.HeaderNames
-
-import scala.collection.JavaConverters._
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
-import scala.util.matching.Regex
-
-abstract class CacheControlFilter extends Filter {
```

Comment:
Deleted CacheControlFilter as I can't see it being used in GitHub.com/hmrc repositories.  Do let me know if deleting it is a problem.